### PR TITLE
Header.

### PR DESCRIPTION
**Pull request by Geljinskiy**
https://todoist.com/app/project/2304123564/task/6450174101

### CODE REVIEW

File: src/components/common/PagesLinks/PagesLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+};
+
+const PagesLinks: React.FC<PagesLinksProps> = ({ pages, className }) => {
+  return (
+    <ul className={`${css.link_list} ${className ?? ''}`}>
+      {pages.map(page => {
+        const { pageName } = page;
+        return (
+          <li key={pageName}>
+            <Link className={css.link_list__link} to={ROUTES[pageName]}>
```
Comment (myroslavsuprun) [inline src/components/common/PagesLinks/PagesLinks.tsx:24]: Don't use JSON for keeping data. A simple variable is enough.
And don't use ROUTES[variable]. Just write the routes inside of that array.

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <header className={css.header}>
+      <Container className={css.header__container}>
+        {windowWidth < 1440 && (
+          <IconButton>
+            <BurgerIcon className={css.burger} />
+          </IconButton>
+        )}
+        <Logo />
+        {windowWidth >= 1440 && <PagesLinks pages={pages} />}
```
Comment (myroslavsuprun) [inline src/components/Header/Header.tsx:21]: Definitely extract view port sizes into variables.

**Review by myroslavsuprun — CHANGES_REQUESTED** [review]

File: src/components/common/PagesLinks/PagesLinks.tsx (hunk 1 above)
Comment (Geljinskiy) [reply src/components/common/PagesLinks/PagesLinks.tsx]: fixed

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (Geljinskiy) [reply src/components/Header/Header.tsx]: fixed

**Review by myroslavsuprun — APPROVED** [review]

**Comment by myroslavsuprun** [comment]
image

**Pull request merged**